roadmap/2022.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap 2022
+
+This is the roadmap planned by the Integration team at MeiliSearch for the year 2022.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2022: the Main Lines
+
+#### Projects
```

Comment:
```suggestion
#### Objectives
```

It looks more like objectives, doesn't it?

Reply:
Today we have around 350 contributors (200+ in 2021). Can it be a quantitative objective to at least double it?

Reply:
Is this number the total of contributors for the integration team?
Do you think the number of contributors is the best metrics to measure if succeeded to work with the community?
But, yes, adding metrics could be nice :)

Reply:
I think it could be discussed in a second time by also including @ferdi05 in the discussion. But when the @meilisearch/devrel-team is working on raising awareness (Stars/Visitors) and growing the community (Community Members / Slack Members). The devrel team has a huge impact on the number of contributors and contributions. The problem is that it's not only a @meilisearch/integration-team work. It's just that when I read the objectives/projects, I feel like there are several that go in that direction.

Reply:
So, should I integrate metrics now, or wait for a discussion before integrating them? :)
In any case, @brunoocasali and I are completely ok with starting using metrics! That would definitely help us understand if we are going in the right direction or not

Reply:
Growing the number of contributors is mostly the @meilisearch/devrel-team responsibility ([it's in our KPIs](https://github.com/meilisearch/devrel/issues/293)), but it benefits from the work done by the Core and Integration teams.
I won't advise making it an objective here as the @meilisearch/integration-team may have a little impact on this number and should not be evaluated accordingly.

Reply:
I'm ok with @ferdi05 :). Keep it like it is.